index.html

```diff
@@ -34,6 +34,7 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
         <option value="other">Other</option>
       </select>
     </div>
+    
 
```

Comment:
Please do not leave more than one empty line between HTML elements. Many newlines make the code not clean